src/test/java/com/redhat/devtools/lsp4ij/features/formatting/JavaServerSideFormatOnCloseBraceTest.java

```diff
@@ -29,17 +29,39 @@ public JavaServerSideFormatOnCloseBraceTest() {
         setTriggerCharacters(";", "\n", "}");
     }
 
-    public void testSimple() {
+    // Simple tests
```

Comment:
Updated the server-side on-type formatting tests to confirm default/enabled behavior vs. explicitly disabled behavior.